nix/buildRocPackage.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs, roc-cli, name, entryPoint, src, outputHash, linker ? "", ... }:
+let
+  packageDependencies = pkgs.stdenv.mkDerivation {
+    inherit src;
+    name = "roc-dependencies";
+    nativeBuildInputs = with pkgs; [ gnutar brotli ripgrep wget ];
+
+    buildPhase = ''
+      list=$(rg -o 'https://github.com[^"]*' ${entryPoint})
+      for url in $list; do
+        path=$(echo $url | awk -F'github.com/|/[^/]*$' '{print $2}')
+        packagePath=$out/roc/packages/github.com/$path
+        mkdir -p $packagePath
+        wget -P $packagePath $url --no-check-certificate
```

Comment:
Is `--no-check-certificate` necessary here?

Reply:
I've removed it and added `cacert` to buildInputs. It still builds just fine so it wasn't necessary.